salt/kubernetes-master/config.jinja

```diff
@@ -22,5 +22,5 @@ KUBE_ALLOW_PRIV="--allow-privileged=true"
 KUBE_ENABLE_DAEMONSETS=true
 
 # How the controller-manager, scheduler, and proxy find the apiserver
-# note: we do not use SSL for local connections
-KUBE_MASTER="--master=http://127.0.0.1:8080"
\ No newline at end of file
+# note: uses haproxy and TLS for HA and resiliency
+KUBE_MASTER="--master=https://api.{{ pillar['internal_infra_domain'] }}:6444"
```

Comment:
we should look at the same logic as other places we use the `{{ api_server_url }}`